src/i_video.c

```diff
@@ -438,7 +496,12 @@ void I_GetEvent(void)
                     I_HandleMouseEvent(&sdlevent);
                 }
                 break;
-
+            // Translate some gamepad events to keys to allow menu navigation
```

Comment:
This seems very hacky. I'd rather see the code in `m_menu.c` extended to support joystick navigation. Indeed, this has already been done somewhat - see #890 and #893 for some prior art.

Reply:
On my mental todo list is port that prior work to heretic,Hexen&strife

Reply:
But I should add @wintermute I'd be very interested to know what is missing from the existing gamepad-menu support from your perspective